mcc/Dispatcher.c

```diff
@@ -159,6 +164,7 @@ static IPTR mNew(struct IClass *cl, Object *obj, struct opSet *msg)
             setFlag(data->flags, FLG_ActiveOnClick);
             setFlag(data->flags, FLG_PasteStyles);
             setFlag(data->flags, FLG_PasteColors);
+            setFlag(data->flags, FLG_HScroll);
```

Comment:
Imagine a TextEditor gadget created without setting a vertical slider to it. Does it scroll vertically with keyboard? It does.
I think we should have the same approach for horizontal scrolling as well. So it should be on by default.
Being able to turn off the horizontal scroll action by setting a NULL for its HorizontalSlider attribute is quite a good idea indeed.